src/platform/backends/shared/qemuimg_process_spec.cpp

```diff
@@ -17,6 +17,8 @@
 
 #include "qemuimg_process_spec.h"
 
+#include <QString>
+#include <QStringList>
```

Comment:
Shouldn't these, strictly, go to the header instead?

Reply:
Right, moved to the header.